### PR TITLE
Leave group command

### LINKED ISSUE #104: Leave group
As a student, I want to be able to leave a group, so that I can join another group of my choice

### PR DESCRIPTION
**Pull request by neyapraveen**
Implement leave group function.
Close #104

### CODE REVIEW

File: src/test/java/seedu/address/ui/PersonCardTest.java [hunk 1]
```diff
@@ -21,14 +21,20 @@ public class PersonCardTest {
```
Comment (neyapraveen) [inline src/test/java/seedu/address/ui/PersonCardTest.java:21]: All tests pass when I run this test file individually but fails when run with the other tests

File: src/main/java/seedu/address/logic/commands/LeaveCommand.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            + PREFIX_EMAIL + "EMAIL "
+            + PREFIX_GROUP + "GROUP NUMBER\n"
+            + "Example: " + COMMAND_WORD + " "
+            + PREFIX_EMAIL + "johnd@u.nus.edu "
+            + PREFIX_GROUP + "1";
+
+    public static final String MESSAGE_LEAVE_SUCCESS = "Leave successful! %1$s\nhas left %2$s!";
+    public static final String MESSAGE_LEAVE_EMAIL_NOT_FOUND = "Person with the provided email not found.";
+    public static final String MESSAGE_LEAVE_GROUP_NOT_FOUND = "Group with the provided group number not found.";
+    public static final String MESSAGE_NOT_IN_GROUP = "The provided person is not a member of the provided group.";
```
Comment (wnchan) [inline src/main/java/seedu/address/logic/commands/LeaveCommand.java:33]: can change to ''This student'' instead

**Review by wnchan — COMMENTED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/LeaveCommand.java (hunk 2 above)
Comment (neyapraveen) [reply src/main/java/seedu/address/logic/commands/LeaveCommand.java]: Done!

**Review by maj0-0 — COMMENTED** [review]
LGTM! Thanks for also working on test cases!

**Pull request merged**